### PR TITLE
localize a11y tab labels in the TabBar

### LINKED ISSUE #13162: Tabs have non-localized string ("Tab N of M")
Search for `// TODO(goderbauer): I10N-ify` in `material/tabs.dart`.

### PR DESCRIPTION
**Pull request by yjbanov**
Fixes https://github.com/flutter/flutter/issues/13162

/cc @HansMuller @goderbauer

### CODE REVIEW

File: packages/flutter_localizations/lib/src/l10n/localizations.dart [hunk 1]
```diff
@@ -116,6 +118,7 @@ class _Bundle_de extends TranslationBundle {
   @override String get pageRowsInfoTitle => r'$firstRow–$lastRow von $rowCount';
   @override String get pageRowsInfoTitleApproximate => r'$firstRow–$lastRow von etwa $rowCount';
   @override String get rowsPerPageTitle => r'Zeilen pro Seite:';
+  @override String get tabLabel => r'$tabIndex von $tabCount';
```
Comment (goderbauer) [inline packages/flutter_localizations/lib/src/l10n/localizations.dart]: Tabulator $tabIndex von $tabCount

**Review by goderbauer — APPROVED** [review]
LGTM

File: packages/flutter_localizations/lib/src/l10n/localizations.dart (hunk 1 above)
Comment (yjbanov) [reply packages/flutter_localizations/lib/src/l10n/localizations.dart]: Done.

**Comment by HansMuller** [comment]
Changes LTGM too.

**Comment by yjbanov** [comment]
Thanks! Will submit on green.

**Pull request merged**